configs/pulpcore/nightly.yaml

```diff
@@ -0,0 +1,64 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: true
+:mash_scripts:
+  - pulpcore-nightly-el8
+:tags:
+  - name: pulpcore-nightly-el8
+    based_off: null
+    helper_tags:
+      pulpcore-nightly-el8-override: null
+    build_target: pulpcore-nightly-el8-build
+    build_package_group_source_tag: pulpcore-nightly-el8-build
+    arches:
+      - x86_64
+    inherits:
+      pulpcore-nightly-el8-build:
+        pulpcore-nightly-el8-override: 0
+      pulpcore-nightly-el8-override:
+        pulpcore-nightly-el8: 0
+    external_repos:
+      - name: centos8-baseos
+        mode: bare
```

Comment:
what is bare, and are we intentionally not using the "flattened" repo?

Reply:
'bare' is a mode added to koji to support modular repositories.  It tells koji to call mergerepo_c with --pkgorigins, which allows normal mergerepo behaviour, while also providing pkgorigins file for koji.

Reply:
As far as which repo we're using, I structured this around the existing pulpcore tag, which is utilizing the individual repos.  We never converted over to the flattened repo for pulpcore, and we haven't had an issue with it (yet)